nncf/quantization/algorithms/weight_compression/scale_estimation.py

```diff
@@ -300,3 +297,18 @@ def apply(
             res[weight_name] = result_scale
 
         return res
+
+
+def get_target_zero_mask(compressed_weights, zp=None):
+    target = compressed_weights
+    if zp is not None:
+        target = target.astype(dtype=zp.dtype) - zp
+    zero_mask = fns.isclose(target, 0)
+    return target, zero_mask
+
+
+def get_near_to_ideal_scale(weight, target, zero_mask, importance):
```

Comment:
Can we change this funny name to something more earthly) For example, `estimate_scales`, `tune_scales`, etc.